setup.py

```diff
@@ -94,7 +94,12 @@
         "train": [
             "tensorflow>=2.4.0",
         ],
-        "waymo": ["waymo-open-dataset-tf-2-4-0"],
+        "waymo": [
+            "waymo-open-dataset-tf-2-4-0",
+            "tabulate==0.8.9",
+            "pathos==0.2.8",
+            "readline==6.2.4.1"
```

Comment:
I am slightly concerned with these dependencies being here. I could suggest moving them out. Then again, `waymo-open-dataset-tf-2-4-0` conflicts with everything anyway.

Reply:
the utility needs `waymo-open-dataset-tf-2-4-0` as well. But i will remove these from `waymo `and update the error message to install these separately.